spec/features/comments_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe "Managing comments" do
+  let!(:article) { create :article, id: 1 }
```

Comment:
I'm fairly confident that setting the `id` won't always take as the database has auto-incrementing ids and might ignore this setter if there's already an article in the test database. I'd suggest removing this, and referencing the article itself where you need to below:
```ruby
visit "/articles/#{article.id}"
```
```ruby
let!(:new_comment) { create :comment, text: 'Test Comment', article: article }
```